src/constants.ts

```diff
@@ -0,0 +1,9 @@
+import { ClientAuthMethod, ResponseType } from 'openid-client'
+
+export const SGID_SIGNING_ALG = 'RS256'
+export const DEFAULT_SGID_CODE_CHALLENGE_METHOD = 'S256'
+export const DEFAULT_SCOPE = 'myinfo.name openid'
+export const SGID_SUPPORTED_FLOWS: ResponseType[] = ['code']
```

Comment:
hrm "flows" is quite generic, I wonder if we can name this variable more specifically? e.g. grant types or response types

Reply:
renamed to `SGID_SUPPORTED_GRANT_TYPES`